templates/class-overview.html

```diff
@@ -49,6 +49,7 @@ <h1>{{ class_info.name }}</h1><span class="cursor-pointer" onclick='hedyApp.rena
                 <button class="green-btn" id="add-student" onclick=$('#add_students_options').toggle();$(this).toggleClass('green-btn');$(this).toggleClass('blue-btn');>{{_('add_students')}}</button>
                 <button class="green-btn" id="customize-class-button" data-cy="customize_class_button" onclick="window.location.href = '/for-teachers/customize-class/{{class_info.id}}'">{{_('customize_class')}}</button>
             {% endif %}
+            <button class="green-btn" id="live_stats_button" onclick="window.location.href = '/live_stats/class/{{class_info.id}}'">{{_('class_live')}}</button>
```

Comment:
class_live I think is a new babel string? Be sure to run the pybabel extraction command (see docs) so it will be added to the pot files and can be translated.